org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -43,6 +39,10 @@
 <unit id="org.eclipse.buildship.feature.group" version="2.1.0.v20170609-1527-m"/>
 <repository location="http://download.eclipse.org/buildship/updates/e47/milestones/2.x/2.1.0.v20170609-1527-m"/>
 </location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
```

Comment:
Can we still keep the lines in their original location? In this way, we only highlight the changes to the version numbers.